make/autoconf/spec.gmk.in

```diff
@@ -889,6 +889,9 @@ USE_EXTERNAL_LIBPNG:=@USE_EXTERNAL_LIBPNG@
 PNG_LIBS:=@PNG_LIBS@
 PNG_CFLAGS:=@PNG_CFLAGS@
 
+ENABLE_LIBSLEEF:=@ENABLE_LIBSLEEF@
+LIBVMATH_CFLAGS:=@LIBVMATH_CFLAGS@
+LIBVMATH_LIBS:=@LIBVMATH_LIBS@
```

Comment:
It's getting better, but you still need to handle the naming here. It should be SLEEF_LIBS and SLEEF_CFLAGS, named after the library you import -- not the library you build. What if some other lib wants to use libsleef?

And you need to untangle the SVE flags (whatever we end doing with that) from LIBVMATH_CFLAGS.